workflows/containerized/main.workflow

```diff
@@ -7,7 +7,7 @@ action "build" {
   runs = "./workflows/containerized/scripts/build.sh"
   env = {
     PEVSL_MAKEFILE_IN = "./workflows/containerized/makeconf/pEVSL_openblas.in"
-    NORMALMODES_MAKEFILE_IN = "./workflows/containerized/makeconf/NormalModes_openblas.in"
+    NORMALMODES_MAKEFILE_IN = "./workflows/containerized/makeconf/NormalModes_mkl.in"
```

Comment:
couple of points:
  * pEVSL can also be compiled against mkl as well
  * we can add a comment here to indicate that if we want to change which libraries are used, we can point to a distinct makefile